.circleci/copy-junit-reports.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+const {execSync} = require('child_process');
+const {join} = require('path');
+const {mkdirSync, rmSync, statSync, readdirSync, copyFileSync} = require('fs');
+
+/**
+ * Discover all test results, which @bazel/jasmine stores as `test.xml` files, in the directory and
+ * return back the list of absolute file paths.
+ */
+const findTestResultsInDir = function (dirPath, files) {
+  for (const file of readdirSync(dirPath)) {
+    const filePath = join(dirPath, file);
+    if (statSync(filePath).isDirectory()) {
+      files = findTestResultsInDir(filePath, files);
+    } else {
+      if (file === 'test.xml') {
```

Comment:
It seems like Bazel does not guarantee this being named like that. Can we use all
XML files? Just an optional comment/question. It does not matter a lot

Reply:
Sure we can just use all XML files, as it looks like they are the only files.